index.js

```diff
@@ -46,13 +47,22 @@ module.exports = function(plugins, options) {
             };
 
             if (node.tag === 'style' && node.content) {
-                var styles = indentResolve([].concat(node.content).join(''), indent);
-                promise = css.process(styles, options)
-                    .then(function(result) {
-                        node.content = [indentResolve(result.css, indent)];
-                    });
+                var includesStandardType = filterType.test('text/css');
+                var hasTypeAttr = node.attrs && node.attrs.type;
+                var isFilterType = hasTypeAttr && filterType.test(node.attrs.type.trim());
+                var isUndefinedType = !hasTypeAttr || includesStandardType && /^(\s+)?$/.test(node.attrs.type);
+                var isStandardCSS = includesStandardType && (isUndefinedType || isFilterType);
+                var isOther = !isStandardCSS && isFilterType;
 
-                promises.push(promise);
+                if (isStandardCSS || isOther) {
```

Comment:
@michael-ciniawsky look a tests 
`<style> <style type> <style type='text/css'>` – postcss process by default
`<style type='filterType/shuold/be/process'>` – process to,
other skip. 

custom attributes another way, because it requires advance markup in html